.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  pull_request:
+
+env:
+  IMAGE_NAME_BACKEND: upetition-backend
+  IMAGE_NAME_FRONTEND: upetition-frontend 
+
+jobs:
+  push:
```

Comment:
A more descriptive name would be nice here

Reply:
something in the ballpark of 'push-to-container-hub', or at least adding an explicit name field for the job? It shows up right now just as 'push' on the 'actions' tab